src/components/banner/Banner.test.tsx

```diff
@@ -0,0 +1,35 @@
+import CloseIcon from '@material-ui/icons/Close';
+import * as enzyme from "enzyme";
+import * as React from "react";
+import * as rest from "../../rest/interface";
+import Banner from './Banner'
+
+describe("Banner", () => {
+
+  const wrapper = enzyme.mount(<Banner client={rest.EvergreenClient("https://evergreen.mongodb.com/api", "https://evergreen.mongodb.com")} message={""} showOptOut={true} onFinishStateUpdate={null} storageKey={"shouldHideBanner"}/>);
```

Comment:
Could you also add a test showing that setting showOptOut=false doesn't show the link?

Reply:
Done